runtime/slot-manager.js

```diff
@@ -17,6 +17,7 @@ let log = !global.document || (global.logging === false) ? () => {} : (...args)
 class SlotManager {
   constructor(domRoot, pec) {
     this._slotBySlotId = new Map();
+    // Map of slot id by particle spec, including particles that have pending requests for a slot.
```

Comment:
`this._slotIdByParticleSpec = new Map();` is as close as you can get in JS to saying literally _map of slot id by particle spec_, so you should leave that out of the comment. Seems like your intent here was to note that the map contains both fulfilled and pending requests, so focus on that.